docs/radio.rst

```diff
@@ -0,0 +1,79 @@
+Radio
+*****
+
+.. py:module:: radio
+
+The ``radio`` module allows devices to work together via simple wireless
+networks.
+
+Functions
+=========
+
+.. py:function:: send(message)
+
+    Sends a message, expressed as a string or bytes.
+
+.. py:function:: recv_bytes()
+
+    Receive the next incoming message on the message queue. Returns ``None`` if
+    there are no pending messages. Messages are returned as bytes.
+
+.. py:function:: receive(fail=False)
+
+    Works in exactly the same way as ``recv_bytes`` but messages are expressed
+    as strings. If the optional ``fail`` argument is ``True``, a ``ValueError``
+    is raised if the incoming message cannot be converted to a string. If the
+    ``fail`` argument is ``False`` (the default) such messages and associated
+    exceptions are silently ignored.
+
+.. py:function:: config(length=32, queue=3, channel=7, power=0, address=b'uBit\x00', data_rate=radio.RADIO_MODE_MODE_Nrf_1Mbit)
+
+    Configures the various settings relating to the radio. The specified
+    default values are sensible.
+
+    The ``length`` defines the size, in bytes, of a message sent via the radio.
+    It can be up to 256 (????? CHECK THIS from memory) bytes long.
```

Comment:
Can someone confirm what this upper limit is? I remember reading it somewhere but can't, for the life of me, find where that's referenced.